### PR TITLE
Added documentation for wharf-provider-azuredevops error responses

### LINKED ISSUE #60: Add documentation for wharf-provider-gitlab error responses
Add documentation for https://github.com/iver-wharf/wharf-provider-azuredevops/pull/14

### PR DESCRIPTION
**Pull request by Alexamakans**
Closes #60

### CODE REVIEW

File: docs/prob/api/provider/invalid-name.md [hunk 1]
```diff
@@ -52,4 +52,4 @@ Iver you would create a new issue over at:
 
 - GitHub provider: <https://github.com/iver-wharf/wharf-provider-github/issues/new>
 - GitLab provider: <https://github.com/iver-wharf/wharf-provider-gitlab/issues/new>
-- Azure DevOps provider: <https://github.com/iver-wharf/wharf-provider-azuredevops/issues/new>
+- Azure DevOps provider: <https://github.com/iver-wharf/wharf-api/issues/new>
```
Comment (jilleJr) [inline docs/prob/api/provider/invalid-name.md:55]: This change seems wrong.

File: docs/prob/_sidebar.md [hunk 2]
```diff
@@ -20,5 +20,20 @@
...
+  - [/prob/provider/azuredevops/getting-build-definition-error](/prob/provider/azuredevops/getting-build-definition-error.md)
+  - [/prob/provider/azuredevops/getting-provider-error](/prob/provider/azuredevops/getting-provider-error.md)
+  - [/prob/provider/azuredevops/getting-token-error](/prob/provider/azuredevops/getting-token-error.md)
+  - [/prob/provider/azuredevops/invalid-event-type-error](/prob/provider/azuredevops/invalid-event-type-error.md)
+  - [/prob/provider/azuredevops/posting-provider-error](/prob/provider/azuredevops/posting-provider-error.md)
+  - [/prob/provider/azuredevops/posting-token-error](/prob/provider/azuredevops/posting-token-error.md)
+  - [/prob/provider/azuredevops/putting-branch-error](/prob/provider/azuredevops/putting-branch-error.md)
+  - [/prob/provider/azuredevops/putting-project-error](/prob/provider/azuredevops/putting-project-error.md)
+  - [/prob/provider/azuredevops/send-trigger-error](/prob/provider/azuredevops/send-trigger-error.md)
+  - [/prob/provider/azuredevops/unmarshal-project-error](/prob/provider/azuredevops/unmarshal-project-error.md)
```
Comment (jilleJr) [inline docs/prob/_sidebar.md:37]: There are a lot of problems here. Maybe you could generalize some of them? Maybe it's not that important to have 4 different problem types for posting/putting project data into the Wharf API

**Review by jilleJr — CHANGES_REQUESTED** [review]

**Review by jilleJr — CHANGES_REQUESTED** [review]
Rejecting until https://github.com/iver-wharf/wharf-provider-azuredevops/pull/14 has been resolved and merged

**Comment by Alexamakans** [comment]
Replaced by #63

**Pull request closed without merging**